docroot/sites/all/themes/ilr_theme/scss/components/_view_modes.scss

```diff
@@ -215,14 +217,21 @@
       @include adjust-font-size-to(36px);
       bottom: .5em;
     }
-    &::before {
-      content: "";
-      position: absolute;
-      top: 0;
-      left: 0;
-      width: 100%;
-      height: 100%;
-      @include background-image(linear-gradient(rgba(#000000, 0), rgba(#000000, 0) 75%, rgba(#000000, .6)));
+  }
+  .field-name-field-image {
+    position: relative;
+    z-index: -1;
```

Comment:
Please add a comment at the end of this line briefly explaining why it's necessary, since it's non-standard